### PR TITLE
React sections i18n

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Finally adds i18n for a bunch of our strings.

The belief is that because these are already strings that already exist in pegasus, crowdin will be smart enough to also get the translations for them in apps (when we do our sync).

### CODE REVIEW

**Comment by joshlory** [comment]
What's the long term plan to merge this with component with the related view on the new Teacher Homepage?  PR https://github.com/code-dot-org/code-dot-org/pull/15938

**Comment by Bjvanminnen** [comment]
There isn't one. Though they show some of the same data, they are also different in that they have different sources of data (that is obviously solvable) and mine has an edit experience.

One can make the argument that it might be worth looking into a common "tabular data" sort of component.

**Comment by tanyaparker** [comment]
@elijah once the strings are in the source files, we'll need to 

1) sync up to crowdin
2) pre-translate the apps files with the pegasus strings
3) sync down
4) then remove strings from pegasus file (I18n gsheet)

I'm worried that if we just sync up/down normally without the pre-translation step we may lose the translations. We need to do the pre-translate step while the strings exist both in pegasus and in apps before we remove them from the pegasus/mobile.yml file.

**Comment by Bjvanminnen** [comment]
What does step 2 entail?

FWIW, I don't have any immediate plans to remove these strings from pegasus. We'll definitely want both versions for a time (while this React code is still behind an experiment). After we make it permanent, we could remove from pegasus, but we'll want to check to see if some of these strings are still used elsewhere in pegasus.

**Comment by tanyaparker** [comment]
Cool, if there is no rush on removing from pegasus then it should be fine. Step #2 is non-engineering work, just clicking through a few things on Crowdin. I'll do this with either Elijah or Josh S.

File: apps/src/templates/teacherDashboard/SectionRow.jsx [hunk 1]
```diff
@@ -134,8 +134,18 @@ class SectionRow extends Component {
...
-  onClickDeleteYes = () => console.log('this is where our delete will happen');
+  onClickDeleteYes = () => {
+    const { section, removeSection } = this.props;
+    $.ajax({
+      url: `/v2/sections/${section.id}`,
+      method: 'DELETE',
+    }).done(() => {
+      removeSection(section.id);
+    }).fail((jqXhr, status) => {
+      // TODO(bjvanminnen): figure out how what we want to do in this case
```
Comment (joshlory) [inline apps/src/templates/teacherDashboard/SectionRow.jsx:145]: TODO

File: apps/src/templates/teacherDashboard/SectionRow.jsx [hunk 2]
```diff
@@ -134,8 +134,18 @@ class SectionRow extends Component {
 
   onClickDeleteNo = () => this.setState({deleting: false});
 
-  // TODO(bjvanminnen)
-  onClickDeleteYes = () => console.log('this is where our delete will happen');
+  onClickDeleteYes = () => {
+    const { section, removeSection } = this.props;
```
Comment (joshlory) [inline apps/src/templates/teacherDashboard/SectionRow.jsx:138]: Test coverage for deleting sections?

Comment (Bjvanminnen) [reply apps/src/templates/teacherDashboard/SectionRow.jsx:138]: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/test/unit/templates/teacherDashboard/teacherSectionsReduxTest.js#L350

File: apps/src/templates/teacherDashboard/SectionRow.jsx (hunk 1 above)
Comment (Bjvanminnen) [reply apps/src/templates/teacherDashboard/SectionRow.jsx:145]: Addressed in upcoming PR https://github.com/code-dot-org/code-dot-org/pull/15941

**Pull request merged**